Added an orange version of the ic_marker pushpin

DeusFigendi opened the pull request:
# Thanks for your contribution.

**Describe the pull request**
@dennisguse asked for an orange pushpin

**Link to the the issue**
#115 

**License agreement**
By opening this pull request, you are providing your contribution under the _Apache License 2.0_ (see [LICENSE.md](LICENSE.md)).
As before: cc0 for this graphic.

dennisguse commented:
Thx!

Sadly, we cannot directly convert the SVG to the Android XML format.
AndroidStudio complains about the linear gradient (there are two: 1st the actual gradient and 2nd the a rotation of the 1st).
And the 2nd gradient cannot be imported.
I am not yet sure how to solve this - AndroidStudio complains about missing stop-infos for the 2nd gradient (which are really not there).

DeusFigendi commented:
Try it now, I just moved the "transform"-informations to the first gradient and removed the second. Then I set the path to use the first gradient.
It renders good in my image viewer and in inkscape and in firefox so I hope Android Studio is okay with it :D

dennisguse commented:
That works like charm!
I will add this properly on the weekend :)

Thanks!
